.github/actions/build-image/action.yaml

```diff
@@ -32,16 +32,19 @@ runs:
       shell: bash
       run: |
         touch dynatrace-operator-bin-sbom.cdx.json
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
```

Comment:
you already have this at line 16-17 (it was only missed in the bootsrapper repo)